test/e2e/data/infrastructure-vsphere-govmomi/v1.9/topology/kustomization.yaml

```diff
@@ -6,4 +6,8 @@ resources:
 patchesStrategicMerge:
   - ../commons/cluster-resource-set-label.yaml
   - ../commons/cluster-network-CIDR.yaml
-  - ../commons/cluster-resource-set-csi-insecure.yaml
\ No newline at end of file
+  - ../commons/cluster-resource-set-csi-insecure.yaml
+patches:
```

Comment:
Is there a reason why we have to do this here with a patch while in 1.10 (& supervisor) it's just directly in the YAML?

Reply:
(up to you)

Reply:
Ah now I got your question.

> If I understand correctly, what you're saying is that the cluster-template-topology.yaml on release-1.10 already had the controlPlanePort while the one in release-1.9 didn't have it and that's why we use a separate patch instead of modifying cluster-template-topology.yaml? (basically to preserve it in it's original state)

That's was the point, but I did only follow it for the cluster template.

> Overall fine to me, just a nit. I was just wondering why it's different :) (especiallly because we also just modify the ClusterClass instead of having a patch there as well)

Fair point :D

I'll change the base file directly instead for consistency then.